src/azure/cli/commands/arm.py

```diff
@@ -226,6 +252,11 @@ def __call__(self, parser, namespace, values, option_string=None):
     cmd = CliCommand(name, handler, simple_output_query=simple_output_query)
     cmd.arguments.update(set_arguments)
     cmd.arguments.update(get_arguments)
+    if function_arguments:
+        cmd.arguments.update(function_arguments)
+    cmd.arguments.pop('instance', None) # inherited from custom_function(instance, ...)
+    cmd.arguments.pop('parent', None)
+    cmd.arguments.pop('expand', None) # possibly inherited from the getter
```

Comment:
Aren't there only about 5 commands that have expand?  If so, I'd vote for manually hiding them

---
In reply to: [76079251](https://github.com/Azure/azure-cli/pull/735#discussion_r76079251) [](ancestors = 76079251,75964122)